diagnostic.md

```diff
@@ -5,74 +5,90 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What is the purpose of a backend?
 
 ```md
-// your response here
+A backend processes and stores data - allows for persistance and shared workload
 ```
 
 Which layer in the MVC pattern is used by the controller to fetch data?
 
 ```md
-// your response here
+Controller layer
 ```
 
 Which layer in the MVC pattern communicates with the model?
 
 ```md
-// your response here
+Model layer
 ```
 
 Why don't we use views in our interpretation of the MVC pattern?
 
 ```md
-// your response here
+Good question, not sure I know why
```

Comment:
Thats okay, we don't use views on the back end because we like doing it on the front end better! It is one of the driving forces behind SPAs.